docs/source/fr/_toctree.yml

```diff
@@ -0,0 +1,84 @@
+- title: "Get started"
+  sections:
+    - local: index
+      title: Home
+    - local: quick-start
+      title: Introduction rapide
+    - local: installation
+      title: Installation
+- title: "Guides"
+  sections:
+    - local: guides/overview
+      title: Vue d'ensemble
+    - local: guides/download
+      title: Télécharger des fichiers
+    - local: guides/upload
+      title: Upload des fichiers
+    - local: guides/cli
+      title: Utiliser le CLI
+    - local: guides/hf_file_system
+      title: HfFileSystem
+    - local: guides/repository
+      title: Dépôts
+    - local: guides/search
+      title: Rechercher
+    - local: guides/inference
+      title: Inférence
+    - local: guides/inference_endpoints
+      title: Inference Endpoints
+    - local: guides/community
+      title: Community Tab
+    - local: guides/collections
+      title: Collections
+    - local: guides/manage-cache
+      title: Cache
+    - local: guides/model-cards
+      title: Model cards
+    - local: guides/manage-spaces
+      title: Gérez vos Spaces
+    - local: guides/integrations
+      title: Intégrer une librairie
+    - local: guides/webhooks_server
+      title: Webhooks
+- title: "Guides conceptuels"
+  sections:
+    - local: concepts/git_vs_http
+      title: Le paradigme Git vs HTTP
+- title: "Référence"
+  sections:
+    - local: package_reference/overview
+      title: Vue d'ensemble
+    - local: package_reference/login
+      title: Connexion et déconnexion
+    - local: package_reference/environment_variables
+      title: Variables d'environnement
+    - local: package_reference/repository
+      title: Gérer les dépôts locaux et en ligne
+    - local: package_reference/hf_api
+      title: API Hugging Face Hub
+    - local: package_reference/file_download
+      title: Téléchargement des fichiers
+    - local: package_reference/mixins
+      title: Mixins & méthodes de sérialisations
+    - local: package_reference/inference_client
+      title: Client d'inférence
+    - local: package_reference/inference_endpoints
+      title: Inference Endpoints
+    - local: package_reference/hf_file_system
+      title: HfFileSystem
+    - local: package_reference/utilities
+      title: Utilitaires
+    - local: package_reference/community
+      title: Discussions et Pull Requests
+    - local: package_reference/cache
+      title: Référence au système de cache
+    - local: package_reference/cards
+      title: Repo Cards et données des Repo Card
+    - local: package_reference/space_runtime
+      title: Space runtime
+    - local: package_reference/collections
+      title: Collections
+    - local: package_reference/tensorboard
+      title: Logger Tensorboard
+    - local: package_reference/webhooks_server
+      title: Serveur Webhook
```

Comment:
We should keep only the files that are added in this PR and leave the others out (or comment them out). Otherwise the docs won't build properly as files will be missing.

Reply:
see error message in CI: https://github.com/huggingface/huggingface_hub/actions/runs/7842688369/job/21401497868?pr=2024

Reply:
Great, thank you for helping, I will keep iterating over other files then.